doc/user/content/sql/functions/date-trunc.md

```diff
@@ -6,19 +6,21 @@ menu:
     parent: 'sql-functions'
 ---
 
-`date_trunc` returns a [`timestamp`](../../types/timestamp) with the "floor value" of the specified time component, i.e. the largest time component less than or equal to the provided value.
+`date_trunc` returns a [`timestamp`] with the "floor value" of the specified time component, i.e. the largest time component less than or equal to the provided value.
```

Comment:
Nitpick: Shouldn't this `timestamp` be replaced with `the same type as ts_val` as well? (or timestamp-like, or something equivalent)

Reply:
Reworded